source/documentation/orgs_spaces_users/index.md

```diff
@@ -4,17 +4,18 @@
 
 An organisation, or org, represents a group of users, applications and environments. Each org shares the same resource, quota and custom domain.
 
-The PaaS team creates the first org for a new project, and assigns at least 1 [org manager](orgs_spaces_users.html#org-manager) to that org. If you want to start a new project on the GOV.UK PaaS, contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk), telling us who the org manager(s) should be.
+The GOV.UK PaaS team creates the first org for a new project, and assigns at least 1 [org manager](orgs_spaces_users.html#org-manager) to that org. If you want to start a new project on the GOV.UK PaaS, contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk), telling us who the org manager(s) should be.
+
+The GOV.UK PaaS is hosted in two locations, London and Ireland. The GOV.UK PaaS team creates all new orgs on the London-based GOV.UK PaaS by default. If you already have an account on the Ireland-based GOV.UK PaaS and request a new org, we will create this new org and account on the London-based GOV.UK PaaS, and you will have two separate accounts.    
```

Comment:
locations -> regions.

Also, now I've come across it several times, "the (London|Ireland)-based GOV.UK PaaS" is really wordy. I think a global change to "the London/Ireland region" would flow significantly better. I'm not going to flag it up in any further comments.